js/packages/teams-ai/src/Application.ts

```diff
@@ -67,26 +70,15 @@ export interface Query<TParams extends Record<string, any>> {
  */
 export interface ApplicationOptions<TState extends TurnState> {
     /**
-     * Optional. Bot adapter being used.
-     * @remarks
-     * If using the `longRunningMessages` option or calling the continueConversationAsync() method,
-     * this property is required.
+     * Optional. Options used to initialize your `BotAdapter`
      */
-    adapter?: BotAdapter;
+    adapter?: BotAdapterOptions;
```

Comment:
The authentication samples will break with this change as it uses the adapter.

Reply:
ya, if we go with this implementation I will open another PR to update the samples once we publish the new version. Do we normally update the samples before we publish?

Reply:
It's better to do a fast-follow for samples to decrease the size of PRs. Also gives a little more time for testing.